bindings/python/test2.py

```diff
@@ -0,0 +1,4 @@
+from tokenizers import Tokenizer
+
+
+tokenizer = Tokenizer.from_file("/home/nicolas/Downloads/tokenizer-wiki.txt")
```

Comment:
Is this file should be added/commited `bindings/python/test2.py`?
If it should be added, it has an absolute path on your 
machine "home/nicolas/Downloads/tokenizer-wiki.txt"

Reply:
I need to remove that file.

Reply:
perfect!